CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+## Prerequisites
+
+- Protocol Buffers compiler `protoc`.
+- Protocol Buffers Go compiler `protoc-gen-go`.
+ 
+Install manually or use `install-proto.sh`.
```

Comment:
please mention that this is for linux, not osx

Reply:
Added.